connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -129,7 +132,7 @@ object functions {
       case v: Array[Byte] => createLiteral(_.setBinary(ByteString.copyFrom(v)))
       case v: collection.mutable.WrappedArray[_] => lit(v.array)
       case v: LocalDate => createLiteral(_.setDate(v.toEpochDay.toInt))
-      case null => unsupported("Null literals not supported yet.")
+      case null => createLiteral(_.setNull(nullType))
```

Comment:
`lag` and `lead` need pass null

Reply:
@LuciferYang can you do me a favor and expand this to support all/most what Literal currently does?

Reply:
ok, let me do this later